### ISSUE TITLE
2.17.1 - can't install this version

### ISSUE THREAD

**Issue by bshlomo** [GeKorm/dart-windows#23]
![2 17 1 install error](https://user-images.githubusercontent.com/42788959/169995383-d1054780-08f4-4e70-b37b-3832f7ec481e.png)

**Comment by GeKorm** [comment]
It looks like it might have been a temporary problem with the Dart server. Are you still having this issue with 2.17.3?

**Comment by bshlomo** [comment]
Yes, I tried again now, and the result is the same. I don't think it is a temporary server problem, because אhe situation is happening the same from my first try with version 2.17.1

**Comment by GeKorm** [comment]
Hmm in the past we've had trouble with the installer using proxies. Would you mind sharing some details like Windows version, whether you're using a proxy, and lastly can you reach this URL please? It should download a zip file (which the updater is failing to download there) https://storage.googleapis.com/dart-archive/channels/stable/release/latest/sdk/dartsdk-windows-x64-release.zip

**Comment by bshlomo** [comment]
I am using this installer for years, without any errors, on the same machine and the same op system, nothing new.
Windows version - win 7 enterprise build 7601 sp1 
Proxy - No
Url - got the file, without any problems
